lib/Bindings/Python/OMModule.cpp

```diff
@@ -55,6 +56,18 @@ struct Object {
     return omEvaluatorObjectValueGetPrimitive(result);
   }
 
+  // Get a list with the names of all the fields in the Object.
+  std::vector<std::string> getFieldNames() {
+    ArrayAttr fieldNames =
```

Comment:
You cannot use `ArrayAttr` in pybind11 code. You **must** use the CAPI for interacting with MLIR objects/classes... 'cause linking issues.